src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -589,21 +605,21 @@ private void setGrids(final GameData data, final String gridType, final String n
     } else if (diagonalConnections.equals("explicit")) {
       diagonalConnectionsImplict = false;
     } else {
-      throw new GameParseException(mapName,
+      throw new GameParseException(mapPathName,
           "diagonal-connections attribute must be either \"explicit\" or \"implicit\"");
     }
-    final int x_size = Integer.valueOf(xs);
-    int y_size;
+    final int xSize = Integer.valueOf(xs);
+    int yySize;                                 // ySize or y_size == checkstyle violation
```

Comment:
`yySize` and `xSize` don't work well as name pairs.
When viewing this code without a diff ` // ySize or y_size == checkstyle violation` will not make very much sense. Why not yet more variations, like `my_ySize`, or int_ySize?

Looking down a few lines, I see:
`map.setGridDimensions(xSize, yySize);`

based on that, some name recommendations:
xSize => mapGridWidth
yySize => mapGridHeight

or
xSize => mapGridColCount
yySize => mapGridRowCount